articles/hosted-pages/login/index.md

```diff
@@ -10,6 +10,8 @@ crews: crew-2
 
 Auth0's Hosted Login Page is the most secure way to easily authenticate users for your applications. The Hosted Login Page is easily customizable right from the [Dashboard](${manage_url}). By default, the Hosted Login Page uses Auth0's [Lock Widget](/libraries/lock) to authenticate your users, but the code of the Hosted Login Page can be customized to replace Lock with the Lock Passwordless widget, or an entirely custom UI can be built in its place, using the [Auth0.js SDK](/libraries/auth0js) for authentication.
 
+The Hosted Login Page is the safest way to authenticate your users with Auth0. Because the Hosted Login Page does not involve embedding authentication in an application, [cross-origin authentication](/cross-origin-authentication) can be avoided.
```

Comment:
The first sentence repeats what we say in the first sentence of the previous paragraph.

Also, I don't get the way this link is presented. It's like you already talked about a problem called `cross-origin authentication` and now you tell them that this way they can avoid them. Isn't the purpose of this link to show this alternative option? Why not use something like `If you do not want to use centralized authentication, you can embed the Lock widget or a custom login form in your application using [cross-origin authentication](/cross-origin-authentication)`

Reply:
Since we're saying HLP is definitively the best and safest option, I was thinking we might want to also show readers the method we're comparing it to (safest vs what?). If that looks bad, and the alternative is something like what you've suggested above, I think I might rather backtrack and not add anything at all here - that reads like COA is a recommended alternative option to me, whereas we would rather recommend that COA wasn't used, wherever possible.

I suppose we could also opt for a slightly different wording, like 

`If you cannot use centralized authentication, you can embed the Lock widget or a custom login form in your application using [cross-origin authentication](/cross-origin-authentication), but be sure to read about its limitations before choosing to do so.` or something similar, that doesn't assume foreknowledge of COA but makes it clear that this isn't the path we'd like them to take.

What do you think?

Reply:
Just went with the latter rewording - let me know if you are not ok with this